pymatgen/analysis/diffraction/tem.py

```diff
@@ -0,0 +1,588 @@
+# coding: utf-8
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+# Credit to Dr. Shyue Ping Ong for the template of the calculator
+
+"""
+This module implements a TEM pattern calculator.
+"""
+
+from __future__ import division, print_function, unicode_literals
+import json
+import os
+from collections import namedtuple
+from fractions import Fraction
+from typing import List, Dict, Tuple, cast
+from functools import lru_cache
+import numpy as np  # type: ignore
+import scipy.constants as sc  # type: ignore
+import pandas as pd  # type: ignore
+import plotly.graph_objs as go  # type: ignore
+import plotly.offline as poff  # type: ignore
+from IPython.display import set_matplotlib_formats  # type: ignore
```

Comment:
This is the offending line; the lines before with `set_matplotlib_formats` and `poff` are entirely unnecessary too, I think this is remnants of debug code